Fix grammar issue (#1883)
act-rules/act-rules.github.io: _rules/html-page-title-descriptive-c4a8a4.md

The `title` elements of embedded documents, such as those in `iframe`, `object`, or `svg` elements, are not applicable because those are not web pages according to the definition in WCAG.

> The [HTML specification - The `title` element](https://html.spec.whatwg.org/#the-title-element) requires that a document only have one `title` element, and that it is a child of the `head` element of a document. However, current HTML specification also describes what should happen in case of multiple titles, and titles outside the `head` element. Because of this, neither of these validation issues causes a conformance problem for WCAG.

Reviewer: I'm struggling with both formulation, likely due to the "that" sounding like a subordinate.
```suggestion
The [HTML specification - The `title` element](https://html.spec.whatwg.org/#the-title-element) requires documents to only have one `title` element; and `title` elements to be children of the `head` element of a document. However, current HTML specification also describes what should happen in case of multiple titles, and titles outside the `head` element. Because of this, neither of these validation issues causes a conformance problem for WCAG.
```
